libvmaf/src/feature/cambi.c

```diff
@@ -132,6 +136,16 @@ static const VmafOption options[] = {
         .min = 0.0001,
         .max = 1.0,
     },
+    {
+        .name = "max_log_contrast",
+        .help = "Max contrast luma level at 10-bits, set to  2^max_log_contrast. "
```

Comment:
I'd say this phrasing is a bit confusing with the comma, maybe it reads better as "Max contrast luma level at 10-bits is set to 2^max_log_contrast."

Reply:
Can we re-use what you had in README?
```
Maximum contrast in log luma level (2^max_log_contrast) at 10-bits. Default 2 is equivalent to 4 luma levels at 10-bit and 1 luma level at 8-bit.
```
Better explicit than sorry.

Reply:
done. Thanks.